### ISSUE TITLE
HumbleBundle option to import only downloadable games

### ISSUE THREAD

**Issue by lkraider** [ssokolow/itad_importer#14]
There are many games not downloadable from HumbleBundle but only offer a key to redeem from services like Steam or Origin, it would be nice to have a way to import only those that are downloadable from the HumbleBundle store, even if they also offer a Steam key, since the download there is usually DRM free.

Below are different types I collected, might not be an exhaustive list. The elements are the innermost ones, there are other outer elements that could be tested for contents.

**Download only:**
![itad-importer-humblebundle-1](https://cloud.githubusercontent.com/assets/52256/8891482/4a6e4ebe-3302-11e5-8942-76a48da0d74b.png)
```html
<div class="js-download-button download-button">
```

---
**Download + Key:**
![itad-importer-humblebundle-2](https://cloud.githubusercontent.com/assets/52256/8891483/56fc29b2-3302-11e5-860d-1eca4b182660.png)
```html
<div class="js-download-button download-button">
(...)
<div class="keyfield redeemed" title="XXX"></div>
```
_example of a redeemed key element. (does not match screenshot)_

---
**Key only (Origin and Steam versions):**
![itad-importer-humblebundle-3](https://cloud.githubusercontent.com/assets/52256/8891489/73112990-3302-11e5-94a3-4afca749e49a.png)
```html
<div title="click to redeem on Origin" class="keyfield ">
click to redeem on Origin
```
---
![itad-importer-humblebundle-4](https://cloud.githubusercontent.com/assets/52256/8891490/74e296fa-3302-11e5-898f-66c0c63fe104.png)
```html
<div title="click to redeem on Steam" class="keyfield ">
click to redeem on Steam
(...)
```
_examples of unredeemed elements._

**Comment by ssokolow** [comment]
Don't worry. I'm very much in favour of only importing stuff that's actually downloadable from GOG.

In fact, it used to work that way but, because of how the new site design works internally, I haven't yet figured out how to extract that data without faking a click to every single entry in the listing. (I decided that half-assed import for the whole-collection view was better than no import at all.)

In the mean time, you can work around it by using the import buttons on individual purchase pages.

**Comment by newroser** [comment]
yes, i think it is best feature

**Comment by ssokolow** [comment]
Yeah, sorry I haven't really had time to work on this in the past year. I've always been a bit shaky on my time management and I've had to put the time I can guarantee toward real-life concerns, like graduating.

At the moment, I'm still working to clear out the massive pile of TODOs that accumulated while I was working on my degree project.
